pkg/virt-controller/services/template.go

```diff
@@ -1157,6 +1163,11 @@ func getRequiredCapabilities(vmi *v1.VirtualMachineInstance) []k8sv1.Capability
 	}
 	// add a CAP_SYS_NICE capability to allow setting cpu affinity
 	res = append(res, CAP_SYS_NICE)
+
+	// add CAP_SYS_ADMIN capability to allow virtiofs
+	if util.IsVMIVirtiofsEnabled(vmi) {
```

Comment:
if hugepages are required, wouldn't we need to ensure hugepage memory is on requests/limits? maybe I'm misunderstanding something here

Reply:
I have this verification in https://github.com/kubevirt/kubevirt/pull/3493/files#diff-8f5063aca22800e362329d88dda6f253R947
Do you think it requires anything else?

Reply:
i think as validation that makes sense for now... forcing the user to request hugepage usage when using virtiofs. 

now that I'm looking at this further, do we need to account for more memory overhead when virtiofs is used in our template.go file? basically, with our current settings do we run the risk of OOMing VMs with virtiofs in some situations?